config/default.json

```diff
@@ -0,0 +1,5 @@
+{
+  "providerUrl": "http://localhost:8545",
+  "network": "kovan",
+  "gasLimit": 6900000
```

Comment:
6500000 is enough .why increase ?

Reply:
i needed to at some point, but if 650000 is working right now thats fine